### PR TITLE
added a background_color option

### PR DESCRIPTION
**Pull request by nicole-a-tesla**
Added an optional param background_color that defaults to '#FFFFFF'

### CODE REVIEW

**Comment by afeld** [comment]
This is great, thanks! Mind adding a note to the README? /cc #9

**Comment by nicole-a-tesla** [comment]
Updated the README!

File: README.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Add the values as a comma-separated list, e.g.
 
 https://sparklines.herokuapp.com/api/v1.png?values=1,4,5,3,2,5
 
+Accepts optional parameters for dot_size, height, line_color, step, and background_color, e.g.
+
+https://sparklines.herokuapp.com/api/v1.png?values=1,4,5,3,2,5;height=100;background_color=FF0000
```
Comment (afeld) [inline README.md]: I think it would be ampersands (`&`) separating the query params, actually.

**Comment by nicole-a-tesla** [comment]
Changed : to &

**Comment by afeld** [comment]
:boom:

**Pull request merged**